test/iriTemplate.test.js

```diff
@@ -132,6 +132,56 @@ describe('middleware/iriTemplate', () => {
       strictEqual(dataset.match(null, fromQuad.predicate, fromQuad.object).size, 1)
       strictEqual(dataset.match(null, toQuad.predicate, toQuad.object).size, 1)
     })
+
+    it('should return individual object for multiple query param occurrences', async () => {
+      let dataset = null
+      const app = express()
+      const tagProp = rdf.namedNode('http://example.org/tag')
+
+      app.use(middleware(iriTemplateMappingBuilder({
+        template: '/{?tag*}',
```

Comment:
This got me by surprise. In actual usage it is not necessary to use the `*` modifier but in the test scenario the handler you not split the values and instead return the `req.params` object like 

```js
{
  tag: 'foo&tag=bar'
}
```

🤷